SCHEMA.md

```diff
@@ -29,6 +30,13 @@ In this document:
 | `OPEN`             | The form is currently accepting responses.                                    |
 | `COLLECT_EMAIL`    | The form should collect the email from submissions. Requires `REQUIRES_LOGIN` |
 | `DISABLE_ANTISPAM` | Disable the anti-spam checks from running on a form submission.               |
+| `WEBHOOK_ENABLED`  | The form should notify the webhook. Has no effect if no webhook is set.       |
+
+### Meta options
+| Field     | Description                                                                                          | Example                                                                                                   |
+| --------- | ---------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
+| `webhook` | Mapping of webhook url and message. Message can use `_USER_MENTION_` to mention the submitting user. | `"webhook": {"url": "https://discord.com/api/webhooks/id/key", "message": "Hello World! _USER_MENTION_"}` |
```

Comment:
I think that we could use something prettier than `_user_mention_` here. How about `{user}`?

Reply:
I've considered making the string formattable and using it as an f-string, but my fear here is that means we allow anyone to run code inside the message. I don't know who will be allowed to create forms yet, so I didn't want to leave an easy exploit in.

Reply:
Yeah, that's a fair concern. We could look at using `string.Formatter` to implement a safe formatter which can access a limited subset of keys, [docs here](https://docs.python.org/3/library/string.html#custom-string-formatting).

Our Modmail bot has a nice implementation of a safe formatter found [here](https://github.com/kyb3r/modmail/blob/56a8c721a96d0c4ad979013a7c49bf51c7cdc848/core/models.py#L156-L182)